File: api/src/main/kotlin/com/gmtkgamejam/models/posts/Availability.kt
package com.gmtkgamejam.models.posts

enum class Availability {

    /**
     * Haven't decided/don't know yet
     */
    UNSURE,

    /**
     * A few hours over the whole jam
     */
    MINIMAL,

    /**
     * Less than 4 hours per day
     */
    PART_TIME,

    /**
     * 4-8 hours per day
     */
    FULL_TIME,

    /**
     * As much time as I can
     */
    OVERTIME,
}

File: ui/src/common/models/availability.ts
import { CustomSelectOption } from "../../pages/jamhome/components/common/CustomSelect.tsx";

export const availability: CustomSelectOption[] = [
  {
    value: "UNSURE",
    label: "Not sure/haven't decided"
  },
  {
    value: "MINIMAL",
    label: "Less than 4 hours per day"
  },
  {
    value: "PART_TIME",
    label: "4 hours per day"
  },
  {
    value: "FULL_TIME",
    label: "8 hours per day"
  },
  {
    value: "OVERTIME",
    label: "More than 8 hours per day"
  },
];
